Dockerfile.experimental

```diff
@@ -1,15 +1,12 @@
 FROM boot2docker/boot2docker
-MAINTAINER Sven Dowideit "SvenDowideit@docker.com"
 
-#DESCRIPTION use the latest experimental build of Docker
+RUN curl -fSL -o /tmp/dockerbin.tgz https://experimental.docker.com/builds/Linux/x86_64/docker-$(cat $ROOTFS/etc/version).tgz && \
+	tar -zxvf /tmp/dockerbin.tgz -C "$ROOTFS/usr/local/bin" --strip-components=1 && \
+	rm /tmp/dockerbin.tgz && \
+	chroot "$ROOTFS" docker -v \
+	sed -ri 's/$/-experimental/' $ROOTFS/etc/version \
```

Comment:
These two lines are missing `&&` in between, so it's probably worth converting this whole file to the "postfix-`&&`" format to ensure we don't miss any (since this file is relatively short, unlike the main `Dockerfile` :sweat:).